test/unit/node_test.rb

```diff
@@ -297,4 +297,14 @@ class NodeTest < ActiveSupport::TestCase
     assert_equal note.likers.length, note.cached_likes
     assert_equal cached_likes-1 , note.cached_likes
   end
+
+  test "return nodes tagged by author and user_id" do
+    jeff = users(:jeff)
+    jeff_notes = nodes(:one)
+    jeff_notes.add_tag('replication:123',jeff)
+    tagged_note = Node.tagged_nodes_by_author('replication:123',2).first
```

Comment:
Here, this is my fault. I had originally said we should put it in `Node`, but then immediately on the next line of #1776 I suggested putting it in `Tag`. I think if you change `Node` to `Tag` this should be fixed. Thank you!!! And apologies!